webapp/channels/src/utils/utils.tsx

```diff
@@ -1643,11 +1643,15 @@ export function getTrackFlowRole() {
     return trackFlowRole;
 }
 
-export function getRoleForTrackFlow() {
-    const startedByRole = TrackFlowRoles[getTrackFlowRole()];
+export const getRoleForTrackFlow = createSelector(
```

Comment:
should we be creating selectors outside of selectors folder?

Reply:
I started to move that around, but I decided not to because there's so much changing here already. I had to hold myself back from trying to fix everything for my own sanity 😅